### DISCUSSION TITLE
Summary on demographic questions

### DISCUSSION THREAD

**Discussion by ChristaGW** [GloBIAS-BioimageAnalysts/Survey_2024#14] [Ideas]
A plot for each: 

- male/female, 
- age, 
- positions (I am not sure how to handle multiple answers best), 
- sector 
- location. 

This would be great for the webpage and the paper. I could also use this for updating the GloBIAS poster to be shown at the EMC in Copenhagen.

**Comment by ChristaGW** [comment]
For Which of these describe your position: a lot of people gave multiple replies. 
Is there a way to bin into the given categories (undergrad/ MSc/ PhD/ Postdoc/ Staff scientist researcher/ Staff scientist core facility (make one here?)/ Core Facility head/ junior group leader/ Professor. With the given order being exclusive (e.g. junior group leader 'excludes' being Facility head, Professor excludes Facilty head and so on?

**Comment by ChristaGW** [comment]
Maybe look at age distribution by gender: 57% male - is this consistent over the age groups?
[survey analysis Google Sheet based.pptx](https://github.com/user-attachments/files/16409952/survey.analysis.Google.Sheet.based.pptx)
 Something like this.

**Comment by bethac07** [comment]
Most of this addressed in #17
